### PR TITLE
image conversion job

### PR DESCRIPTION
**Pull request by jjidt**
- Adds an image conversion worker 
- Currently just logs the results to the console so you can get the processed-image urls
- Attempted to do this with streaming, but sharp loads the data into a buffer to extract metadata, so that wasn't giving us any advantage with memory (also it was buggy)

Next Steps:
- create field structure to return formats
- update field after job runs (the job will probably need an admin route to update fields), I'm currently passing the document and field id through the job queue to make this happen
- strip metadata on the original image, can use sharp library on the api to do this
- attach our metadata to the field item

### CODE REVIEW

File: workers/src/coordinators/imageProcessing.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  // webPHalf
+  const webPHalf = await image
+    .resize(Math.round(metadata.width / 2))
+    .toFormat('webp')
+    .toBuffer()
+
+  // pngFull
+  const pngFull = await image
+    .toFormat('png')
```
Comment (dheniges) [inline workers/src/coordinators/imageProcessing.js]: I think we are going with `avif` not `png` as the 2nd format

**Comment by jjidt** [comment]
updated

File: workers/src/coordinators/imageProcessing.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    .toBuffer()
+
+  // webPHalf
+  const webPHalf = await image
+    .resize(Math.round(metadata.width / 2))
+    .toFormat('webp')
+    .toBuffer()
+
+  // pngFull
+  const pngFull = await image
```
Comment (jcontonio) [inline workers/src/coordinators/imageProcessing.js]: Change var names to avif

**Comment by jjidt** [comment]
updated

**Review by jcontonio — APPROVED** [review]

**Pull request merged**